web/src/views/InvestigationDetail.vue

```diff
@@ -151,6 +151,15 @@ interface StackFrame {
   treeDepth: number;
 }
 
+interface BandSpec {
+  frame: number;
+  palette: string[];
```

Comment:
```suggestion
  palette: string;
```

To make compatible with my other comment